### PR TITLE
Show load and file_extensions in accessors

### PR DESCRIPTION
**Pull request by oxinabox**
Like #26 this seemed an interesting intellectual exercise.
This functionality looks scary and technically it is internals.
But I have been using it in ExprTools.jl in like every version of julia back til 1.0
Having to use an internal method for this is another manifestation of https://github.com/JuliaLang/julia/issues/20555

I put this in a helper function so that after #26 we could add it to that list also

### CODE REVIEW

**Comment by exaexa** [comment]
OK actually to keep people from getting confused, could we make the `type_accessors` a complete standalone "visible" method? It is a bit different from accessors (they don't "eat" an actual object).

Other than that this looks wunderbar, thanks again!

PS I see you rebased, perfect. :D

**Comment by oxinabox** [comment]
I think it is pretty clear from how it is printed:

```julia

julia> AbstractFBCModels.required_accessors()
[1] reactions(a::AbstractFBCModels.AbstractFBCModel) @ AbstractFBCModels none:0
[2] n_reactions(a::AbstractFBCModels.AbstractFBCModel) @ AbstractFBCModels none:0
[3] metabolites(a::AbstractFBCModels.AbstractFBCModel) @ AbstractFBCModels none:0
...
[11] save(a::AbstractFBCModels.AbstractFBCModel, path::String) @ AbstractFBCModels none:0
[12] load(::Type{A}, path::String) where A<:AbstractFBCModel @ AbstractFBCModels ~/repos/AbstractFBCModels.jl/src/io.jl:7
[13] filename_extensions(::Type{A}) where A<:AbstractFBCModel @ AbstractFBCModels ~/repos/AbstractFBCModels.jl/src/io.jl:25
```

but up to you.
Its getting late here so I will leave it with you, if that ok?
Feel free to push to this branch before you merge.

**Comment by exaexa** [comment]
actually yeah with the types it looks completely transparent. Thanks a lot!

File: src/utils.jl [hunk 1]
```diff
@@ -48,9 +43,29 @@ function accessors()
             methodswith(AbstractFBCModels.AbstractFBCModel, f, ms)
         end
     end
+
+    append!(ms, _type_accessors())
+    return ms
+end
+
+function _type_accessors()
+    # special case these as they take the type not the instance
```
Comment (exaexa) [inline src/utils.jl:52]:
```suggestion
    # special case: some "accessors" take the Type argument instead of actual instance
```

**Pull request merged**